### DISCUSSION TITLE
Milvus 1.x - IVF PQ index

### DISCUSSION THREAD

**Discussion by durai89** [milvus-io/milvus#18639] [Q&A and General discussion]
We are using Milvus 1.x and collection is bit big hence we use IVF PQ index. There are recommendations around configuring `nlist` as `4 * square root of no. of entities in a segment`, whereas would like to know if there are any recommendations around optimal product quantization factor parameter `m`?

**Comment by yhmo** [comment]
Nope, we have no recommendation for the parameter 'm'. It is not easy to give a recommended value for this parameter.

**Reply by durai89** [reply]
Thanks!, Can I say higher the value of "m", better is the recall? So, lowering "m" to reduce memory footprint may affect the quality of results? Or it will not cause a great impact on the quality?

**Reply by yhmo** [reply]
- dimension must be able to divided by "m"
- increase the "m",  the recall is better, but search slower.
- decrease the "m", recall is worse, search faster.
